src/Vehicle/FactGroups/VehicleGPSFactGroup.cc

```diff
@@ -10,6 +10,8 @@
 #include "VehicleGPSFactGroup.h"
 #include "Vehicle.h"
 #include "QGCGeo.h"
+#include "QGCLoggingCategory.h"
+#include "development/mavlink_msg_gnss_integrity.h"
```

Comment:
So QGC is indeed building and shipping development messages? I thought recently we decided not to do that anymore due to the standard modes fiasco? Should we put the mavlink development message features behind a preprocessor guard for debug builds only?

Reply:
I think it was like that, and then we needed development for some particular reason that I can't recall right now